recipes/django-parler/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "django-parler" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f633522c06727852978cf2f92c94a98d900dff47235412e64ed46d63e68b1f09
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - django >=1.7
```

Comment:
`six` is indeed a required dependency, as listed in [django-parler's `setup.py`](https://github.com/django-parler/django-parler/blob/126b2a5bb96c44bdddbf7a620477b19e44bfa8ca/setup.py#L41) so you should have it here under `run`.

Reply:
True this, resolved now.